float32 meters
---
string csum
